lib/DataQueue.js

```diff
@@ -0,0 +1,207 @@
+const { ProgramCall } = require('./ProgramCall');
+const { CommandCall } = require('./CommandCall');
+const { xmlToJson } = require('./utils');
+
+// QRCVDTAQ Parameters
```

Comment:
Interested in what others think about pulling Parameter information out into a structure like this. Might work a little better after the this PR drops: https://github.com/IBM/nodejs-itoolkit/pull/139